src/util/storybook-docs/knobHelpers.ts

```diff
@@ -4,7 +4,7 @@
  * @param {Array} values list of possible prop values
  * @returns {Object} options object for storybook knobs
  */
-export const arrayToOptions = (values) =>
+export const arrayToOptions = (values: Array<any>): object =>
```

Comment:
heads up: this file is going to be removed the moment we migrate away from using knobs.

Reply:
no harm then.